index.html

```diff
@@ -50,6 +58,203 @@ <h1>Hey there.
                 <img src="./pic/Illustration.png" alt="Illustration">
             </div>
         </section>
+
+
+
+        <section class="works">
+            <div class="title">
+                <h1 id="workstitle">My Recent Works</h1>
```

Comment:
- [ ] I noticed here the font-weight and font family are not the same as the Figma template kindly update it to be the same. 👌🏻